cactus/plugin/builtin/pagecontext.py

```diff
@@ -0,0 +1,120 @@
+#coding:utf-8
+import logging
+import re
+
+
+logger = logging.getLogger(__name__)
+
+
+class PageContextPlugin(object):
+    """
+    A plugin to extract context variables from the top of the file
+
+    TODO:
+    - read style of metadata from config
+    - read splitChar from config
```

Comment:
In the current implementation `splitChar` is passed as a parameter, in this new version it is hard coded. Putting it into the config looks like a middle ground to me. But I'm also not sure, if it is necessary.